### PR TITLE
Add default home layout.

### PR DESCRIPTION
**Pull request by knightburton**
- Add a default layout as home page in case of unselected device.
- Load the default home in case of unsupported device.
- Fetch the last result of the measures in the default view for each target.
- Change the device selector function.
- Add a loading layer to the device pages.

JSRemoteTest-DCO-1.0-Signed-off-by: Imre Kiss kissi.szeged@partner.samsung.com

### CODE REVIEW

**Comment by knightburton** [comment]
I've created the new home page layout, this PR is changes a few things and you can check these changes on my gh-page: https://knightburton.github.io/js-remote-test/

File: index.html [hunk 1]
```diff
@@ -44,65 +45,89 @@
...
     </div>
-    <div class="col-md-6">
-      <div id="binary-chart"></div>
-    </div>
-    <div class="col-md-6">
-      <div id="memory-chart"></div>
+  </div>
+
+  <div id="target-info-placeholder">
+    <p>Loading the device informations...</p>
```
Comment (rtakacs) [inline index.html]: informations --> information

File: index.html [hunk 2]
```diff
@@ -44,65 +45,89 @@
...
+          <h1>IoT.js</h1>
+          <p class="lead">Automated correctness and performance test results</p>
+        </div>
+        <div class="project-overview">
+          <p>The purpose of the project is to run the official IoT.js test-suite on different platforms. The testing happens once a day (at UTC 17:00) using the latest master.</p>
+          <p>Since IoT.js focuses on low footprint devices, the run-time memory consumption is tracked during the test execution. This means that each passed tests have information about their peak usage of the stack and the heap memory areas. The size of the compiled IoT.js is also saved.</p>
+          <p>The collected values are visualized on charts that help to observe how IoT.js evolves day by day. If you are interested in a platform, please choose one of the options below.</p>
+        </div>
+        <div class="project-targets" id="home-targets">
+          <p class="text-center targets-placeholder">Loading target informations...</p>
```
Comment (rtakacs) [inline index.html]: informations --> information

**Comment by rtakacs** [comment]
Thanks @knightburton for this PR. LGTM.

**Comment by knightburton** [comment]
@rtakacs I've updated the PR based on your review.

**Comment by robertsipka** [comment]
LGTM

**Pull request merged**